docs/csharp/language-reference/operators/relational-operators.md

```diff
@@ -0,0 +1,74 @@
+  - "<= operator [C#]"
+  - "greater than or equal to operator [C#]"
+  - ">= operator [C#]"
+---
+# Relational operators (C# Reference)
+
+The [`<` (less than)](#less-than-operator-), [`>` (greater than)](#greater-than-operator-), [`<=` (less than or equal)](#less-than-or-equal-operator-), and [`>=` (greater than or equal)](#greater-than-or-equal-operator-) relational operators compare their operands. Those operators support all [integral](../keywords/integral-types-table.md) and [floating-point](../keywords/floating-point-types-table.md) numeric types.
+
+> [!NOTE]
+> For equality and relational operators `==`, `>`, `<`, `>=`, and `<=`, if any of the operands is not a number (<xref:System.Double.NaN?displayProperty=nameWithType> or <xref:System.Single.NaN?displayProperty=nameWithType>), the result of operation is `false`. That means that the `NaN` value is neither greater than, less than, nor equal to any other `double` (or `float`) value, including `NaN`. For more information and examples, see the <xref:System.Double.NaN?displayProperty=nameWithType> or <xref:System.Single.NaN?displayProperty=nameWithType> reference article.
+
+Enumeration types also support relational operators. For operands of the same [enum](../keywords/enum.md) type, the corresponding values of the underlying integral type are compared.
+
+The [`==` and `!=` operators](equality-operators.md) check if their operands are equal or not.
+
+## Less than operator \<
+
+The `<` operator returns `true` if its first operand is less than its second operand, `false` otherwise:
+
+[!code-csharp-interactive[less than example](~/samples/snippets/csharp/language-reference/operators/GreaterAndLessOperatorsExamples.cs#Less)]
+
+## Greater than operator >
+
+The `>` operator returns `true` if its first operand is greater than its second operand, `false` otherwise:
+
+[!code-csharp-interactive[greater than example](~/samples/snippets/csharp/language-reference/operators/GreaterAndLessOperatorsExamples.cs#Greater)]
+
+## Less than or equal operator \<=
+
+The `<=` operator returns `true` if its first operand is less than or equal to its second operand, `false` otherwise:
+
+[!code-csharp-interactive[less than or equal example](~/samples/snippets/csharp/language-reference/operators/GreaterAndLessOperatorsExamples.cs#LessOrEqual)]
+
+## Greater than or equal operator >=
+
+The `>=` operator returns `true` if its first operand is greater than or equal to its second operand, `false` otherwise:
+
+[!code-csharp-interactive[greater than or equal example](~/samples/snippets/csharp/language-reference/operators/GreaterAndLessOperatorsExamples.cs#GreaterOrEqual)]
+
+## Operator overloadability
+
+A user-defined type can [overload](../keywords/operator.md) the `<`, `>`, `<=`, and `>=` operators.
+
+If a type overloads one of the `<` or `>` operators, it must also overload another one. If a type overloads one of the `<=` or `>=` operators, it must also overload another one.
```

Comment:
```suggestion
If a type overloads one of the `<` or `>` operators, it must also overload both `<` and `>`. If a type overloads one of the `<=` or `>=` operators, it must also overload both `<=` and `>=`.
```

I think that's a bit more clear.

Reply:
@BillWagner thanks; I didn't apply the suggestion, but addressed the feedback almost word in word.